### ISSUE TITLE
error on newly installed version of vtop

### ISSUE THREAD

**Issue by ishanSrt** [MrRio/vtop#124]
```
evalmachine.<anonymous>:2
vtop@0.5.7 | MIT | deps: 10 | versions: 47
^

SyntaxError: Invalid or unexpected token
    at new Script (vm.js:74:7)
    at createScript (vm.js:246:10)
    at Object.runInNewContext (vm.js:291:10)
    at safeEval (/usr/local/lib/node_modules/vtop/node_modules/safe-eval/index.js:13:6)
    at /usr/local/lib/node_modules/vtop/upgrade.js:25:24
    at ChildProcess.exithandler (child_process.js:273:7)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:957:16)
    at Socket.stream.socket.on (internal/child_process.js:378:11)
    at Socket.emit (events.js:182:13)
```

**Comment by Eeems** [comment]
Duplicate of #121

**Comment by MrRio** [comment]
This should now be fixed in v0.6.0 👍
